packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -12,12 +13,18 @@ import 'raised_button.dart';
 /// An end-aligned row of buttons.
 ///
 /// Places the buttons horizontally according to the padding in the current
-/// [ButtonTheme]. The children are laid out in a [Row] with
+/// [ButtonBarTheme]. The children are laid out in a [Row] with
 /// [MainAxisAlignment.end]. When the [Directionality] is [TextDirection.ltr],
 /// the button bar's children are right justified and the last child becomes
 /// the rightmost child. When the [Directionality] [TextDirection.rtl] the
 /// children are left justified and the last child becomes the leftmost child.
 ///
+/// The [ButtonBar] can be configured with a [ButtonBarTheme]. For any property
+/// not set on the [ButtonBar] directly it will look for a surrounding
```

Comment:
This is a complicated sentence. Maybe instead of "directly it will look" => the button bar will look

Note also: sometimes it useful to use the link notation for the first reference and then just write out the class name for subsequent references. The font changes implied by [Foo] can make the text harder to read.

Reply:
Good to know. Is it just link the first reference in the entire doc string or for each paragraph or sentence?